tests/unit/Stubs/EmailOnlyAuthenticator.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace SilverStripe\RestfulServer\Tests\Stubs;
+
+use SilverStripe\Dev\TestOnly;
+use SilverStripe\Security\Member;
+
+/**
+ * An unsecure test authenticator.
+ */
+class EmailOnlyAuthenticator implements TestOnly
+{
+    /**
+     * @return Member|false
+     */
+    public static function authenticate()
+    {
+        //if there is no username or password, fail
+        if (!isset($_SERVER['PHP_AUTH_USER'])) {
+            return null;
+        }
+
+        /** @var null|Member $member */
+        $member = Member::get()->find('Email', $_SERVER['PHP_AUTH_USER']);
```

Comment:
I don't think this is a very good way to perform the test because we could see tests passing that shouldn't if (for example) the first authenticator has a bug that means passwords are not validated properly.

You could either create an actual mock instance of an authenticator (assuming there's an interface to mock) *or* create some kind of hard-coded token authenticator (eg: if AUTH_USER === 'SOME-TOKEN' return a user).

Ideally this user will be a new user added to the fixtures too.